packages/js/manifests/polywrap/src/formats/polywrap.app/validate.ts

```diff
@@ -12,6 +12,7 @@ import {
 import AppManifestSchema_0_1_0 from "@polywrap/polywrap-manifest-schemas/formats/polywrap.app/0.1.0.json";
 import AppManifestSchema_0_2_0 from "@polywrap/polywrap-manifest-schemas/formats/polywrap.app/0.2.0.json";
 import AppManifestSchema_0_3_0 from "@polywrap/polywrap-manifest-schemas/formats/polywrap.app/0.3.0.json";
+import AppManifestSchema_0_4_0 from "@polywrap/polywrap-manifest-schemas/formats/polywrap.app/0.4.0.json";
```

Comment:
Why are we adding version 0.4.0 of app, plugin, and wrap manifests? I don't see the need.